lib/platform/github/index.ts

```diff
@@ -699,43 +699,60 @@ function matchesState(state: string, desiredState: string): boolean {
   return state === desiredState;
 }
 
+type GithubPullsResponse = {
+  number: number;
+  head: {
+    ref: string;
+    sha: string;
+    repo: { full_name: string };
+    user: { login: string };
+  };
+  title: string;
+  state: string;
+  merged_at: string;
+  created_at: string;
+  closed_at: string;
+}[];
+
 export async function getPrList(): Promise<Pr[]> {
   logger.trace('getPrList()');
   if (!config.prList) {
     logger.debug('Retrieving PR list');
-    let res;
+    let prList: GithubPullsResponse;
     try {
-      res = await githubApi.getJson<{
-        number: number;
-        head: { ref: string; sha: string; repo: { full_name: string } };
-        title: string;
-        state: string;
-        merged_at: string;
-        created_at: string;
-        closed_at: string;
-      }>(
-        `repos/${
-          config.parentRepo || config.repository
-        }/pulls?per_page=100&state=all`,
-        { paginate: true }
-      );
+      prList = (
+        await githubApi.getJson<GithubPullsResponse>(
+          `repos/${
+            config.parentRepo || config.repository
+          }/pulls?per_page=100&state=all`,
+          { paginate: true }
+        )
+      ).body;
     } catch (err) /* istanbul ignore next */ {
       logger.debug({ err }, 'getPrList err');
       throw new ExternalHostError(err, PLATFORM_TYPE_GITHUB);
     }
-    config.prList = res.body.map((pr) => ({
-      number: pr.number,
-      branchName: pr.head.ref,
-      sha: pr.head.sha,
-      title: pr.title,
-      state:
-        pr.state === PrState.Closed && pr.merged_at?.length
-          ? /* istanbul ignore next */ PrState.Merged
-          : pr.state,
-      createdAt: pr.created_at,
-      closed_at: pr.closed_at,
-      sourceRepo: pr.head?.repo?.full_name,
-    }));
+    config.prList = prList
+      .filter((pr) => {
+        try {
+          return pr.head.user.login === config.renovateUsername;
```

Comment:
Am I using this right? https://developer.github.com/v3/pulls/#get-a-pull-request

Reply:
Try fetching any public repo, like renovate?

Reply:
Looks okay:
`pr.title === "fix(github): Return only PRs that was created by Renovate"`
`pr.head.user.login ===  "zharinov"`
`pr.base.user.login === "renovatebot"`